Smt/Tactic/Smt.lean

```diff
@@ -116,6 +117,7 @@ private def addDeclToUnfoldOrTheorem (thms : Meta.SimpTheorems) (e : Expr) : Met
 
 open Reconstruction.Certifying in
 def rconsProof (hints : List Expr) : TacticM Unit := do
+  evalTactic (← `(tactic| repeat rewrite [Smt.Util.iff_eq_eq] at *))
```

Comment:
I am thinking of changing the return type of `rconsProof` from `TacticM Unit` to `MetaM (List MVarId)`. Could you try to replace this line with code that does not use the tactic monad?